apps/earn/components/AddSection/AddSectionLegacy.tsx

```diff
@@ -90,7 +90,12 @@ export const AddSectionLegacy: FC<{ pool: Pool }> = ({ pool: _pool }) => {
             }
           >
             <Checker.Network fullWidth size="xl" chainId={_pool.chainId}>
-              <Checker.Amounts fullWidth size="xl" chainId={_pool.chainId} amounts={[parsedInput0, parsedInput1]}>
+              <Checker.Amounts
+                fullWidth
+                size="xl"
+                chainId={_pool.chainId as ChainId}
+                amounts={[parsedInput0, parsedInput1]}
```

Comment:
assign this array to a variable in a useMemo or this will rerun every render

Reply:
I haven't changed anything about the logic, so that means it's running every render now as well?